core/src/main/scala/chisel3/Aggregate.scala

```diff
@@ -268,6 +278,40 @@ sealed class Vec[T <: Data] private[chisel3] (gen: => T, val length: Int) extend
       a <> b
   }
 
+  /** Bi-directional strict connection from a producer to consumer: consumer :<>= producer
+    *
+    * @note Leaf fields aligned in relative direction to consumer will be assigned to from the corresponding producer field
+    * @note Leaf fields flipped in relative direction to producer will be assigned to from the corresponding consumer field
+    * @note the Chisel type of this and that must be equivalent.
+    *  - All ground types are the same (UInt and UInt are same, SInt and UInt are not), but widths can be different
+    *  - All vector types are the same length
+    *  - All bundle types have the same field names and field flips
+    * @note the leaf fields that are ultimately assigned to, must be assignable. This means they cannot be module inputs or instance outputs.
+    * @note the length of this Vec must match the length of the input Seq
+    */
+  def :<>=(that: Seq[T])(implicit sourceInfo: SourceInfo, moduleCompileOptions: CompileOptions): Unit = {
+    if (this.length != that.length) {
+      Builder.error("Vec and Seq being bulk connected have different lengths!")
+    }
+    for ((a, b) <- this.zip(that))
+      a :<>= b
+  }
+
+  /** Bi-directional strict connection from a producer to consumer: consumer :<>= producer
+    *
+    * @note Leaf fields aligned in relative direction to consumer will be assigned to from the corresponding producer field
+    * @note Leaf fields flipped in relative direction to producer will be assigned to from the corresponding consumer field
+    * @note the Chisel type of this and that must be equivalent.
+    *  - All ground types are the same (UInt and UInt are same, SInt and UInt are not), but widths can be different
+    *  - All vector types are the same length
+    *  - All bundle types have the same field names and field flips
+    * @note the leaf fields that are ultimately assigned to, must be assignable. This means they cannot be module inputs or instance outputs.
+    * @note the length of this Vec must match the length of the input Seq
+    */
```

Comment:
Is it possible to use `@inheritdoc` or [`@define`](https://docs.scala-lang.org/overviews/scaladoc/for-library-authors.html) to DRY out the documentation here?

Reply:
I refactored a bunch of the doc strings here and made sure they were @croup Connect-ed, but

`@inheritdoc` doesn't work because these are not actually overriding anything, they are just more functions

`@define` doesn't help much because these are complicated formatted chunks of text and it doesn't seem to work well for that

Also TIL putting a bunch of @note in series gets formatted as a single `Note` with commas separating all the strings, not the most helpful formatting, so I changed this to bulleted list. (At least the default `sbt doc` HTML)